### PR TITLE
LIME-219: * Theme switch general layout

### LINKED ISSUE #219: BUG: Button for switching dark/light mode is absent on common layout
### Describe your issue

Сommon layout doesn't include button for switching dark/light mode

### Steps to reproduce

1. Open http://lime.eu-north-1.elasticbeanstalk.com/
2. Being redirected to Sign In page, switch into light mode
3. Login with valid credentials.
4. Being redirected to Home page, check if 

- light mode is saved in Local storage
- page is displayed in light mode
- button for switching dark/light mode is present in the right bottom corner

### What was the actual result?

Page is displayed in dark mode, layout doesn't include button for switching dark/light mode

### What was the expected result?

Page is displayed in light mode, layout includes button for switching dark/light mode

### Put here any screenshots or videos (optional)

image

Image

### What OS are you seeing the problem on?

Windows

### What browsers are you seeing the problem on?

Chrome

### Severity

Major (S3)

### LINKED ISSUE #267: BUG: Sidebar elements are not clickable and don't navigate to corresponding page
### Describe your issue

Sidebar elements are not clickable and don't navigate to corresponding pages

### Steps to reproduce

1. Open login page http://live-in-motion-everyday.com/sign-in 
2. Login with valid credentials.
3. Being redirected to Homepage, check if sidebar elements are clickable and provide navigation

### What was the actual result?

Sidebar elements are not clickable and don't navigate to corresponding pages

### What was the expected result?

Sidebar elements are clickable and navigate to corresponding pages

### Put here any screenshots or videos (optional)

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/123622110/3b275095-141b-4fe3-9146-0c1469953dee

### What OS are you seeing the problem on?

Windows

### What browsers are you seeing the problem on?

Chrome

### Severity

Major (S3)

### LINKED ISSUE #277: BUG: Location of button to switch light/dark mode on 404 page doesn't correspond to other pages
### Describe your issue

Button to switch light/dark mode on 404 page is located in the middle of the page, not corresponding to the design of other pages

### Steps to reproduce

1. Open page https://live-in-motion-everyday.com/
2. Being redirected to Sign in page, pay attention to location of button to switch light/dark mode 
3. Enter non-existing path e.g. https://live-in-motion-everyday.com/jjj
4. Being redirected to 404 page, compare location of button to switch light/dark mode with Sign In
### What was the actual result?

Button to switch light/dark mode on 404 page is located in the middle of the page, not corresponding to the design of other pages

### What was the expected result?

Button to switch light/dark mode is located in the right bottom corner, corresponding to the design of other pages

### Put here any screenshots or videos (optional)

image

### What OS are you seeing the problem on?

Windows

### What browsers are you seeing the problem on?

Chrome

### Severity

Minor (S4)

### LINKED ISSUE #295: BUG: The labels are not displayed on the Sign In/Up pages after switching to the light mode
### Describe your issue

The labels for email and password are not displayed on the Sign In/Up and Reset password pages after switching to the light mode

### Steps to reproduce

1. Go to the [https://live-in-motion-everyday.com/sign-in](https://live-in-motion-everyday.com/sign-in)
2. Switch to the light mode
3. Pay attention to the labels above fields
4. Click on the "Create an Account" option
5. Pay attention to the labels above fields

### What was the actual result?

The labels for email and password are not displayed on the Sign In/Up  and Reset password pages after switching to the light mode

### What was the expected result?

The labels for email and password are displayed on the Sign In/Up and Reset password pages after switching to the light mode

### Put here any screenshots or videos (optional)

Image

Image

Image

Image

### What OS are you seeing the problem on?

Windows, Android

### What browsers are you seeing the problem on?

Chrome

### Severity

Minor (S4)

### LINKED ISSUE #335: Make switch theme icon more visible on sign in/up pages
Image

We need to make the icon more visible against the background

### PR DESCRIPTION
**Pull request by DavidCharif**

### CODE REVIEW

**Comment by DavidCharif** [comment]
image
image


image
image
image

**Comment by DavidCharif** [comment]
Testing

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/0247e354-08c8-44d0-a3de-d0e917479e40

**Review by valery-chumak — DISMISSED** [review]

File: frontend/src/bundles/common/components/theme-switcher/theme-switcher.tsx [hunk 1]
```diff
@@ -13,7 +14,11 @@ type Properties = {
     className?: string;
 };
 
-function ThemeSwitcher({ className }: Properties): JSX.Element {
+const defaultClassName = 'fixed md:bottom-4 md:right-4 bottom-1 right-1';
+
+function ThemeSwitcher({
```
Comment (anton-otroshchenko) [inline frontend/src/bundles/common/components/theme-switcher/theme-switcher.tsx:19]:
```suggestion
function handleThemeSwitcher({
```

Comment (anton-otroshchenko) [reply frontend/src/bundles/common/components/theme-switcher/theme-switcher.tsx:19]: A10 https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]

**Comment by DavidCharif** [comment]
Update testing

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/7b18b06d-d7bd-4d98-9811-9d01c9268360

**Comment by nikita-remeslov** [comment]
Please double check all components
image

your version
image

**Comment by DavidCharif** [comment]
image
image

**Comment by DavidCharif** [comment]
Update

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/a7739d5b-6dea-44ea-83b5-182775a8358a

**Comment by DavidCharif** [comment]
image
image

**Comment by DavidCharif** [comment]
image
image

**Review by valery-chumak — DISMISSED** [review]

**Comment by DavidCharif** [comment]
Update: 
Before
image

After
image
image

**Review by nikita-remeslov — DISMISSED** [review]

**Comment by valery-chumak** [comment]
@DavidCharif 
The Logout button on light theme is not visible
image

**Comment by valery-chumak** [comment]
Could you also please change the colors for radio buttons in profile settings for light theme? Because it is not changing at all
image

**Comment by DavidCharif** [comment]
image
image
Fix radio buttons and logout

**Review by valery-chumak — DISMISSED** [review]

**Comment by DavidCharif** [comment]
Before
image
image

After
image
image
image
image

**Comment by stpalamar** [comment]
Can you please make this text the same for the sign-up and sign-in forms?
image
image

**Comment by DavidCharif** [comment]
image
image

**Comment by DavidCharif** [comment]
image
image

**Comment by DavidCharif** [comment]
image
image
image

**Review by stpalamar — DISMISSED** [review]

**Comment by DavidCharif** [comment]
image
image
image
image

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**